webpack.config.js

```diff
@@ -97,7 +97,7 @@ jsLoader = {
 };
 
 if ( CALYPSO_ENV === 'development' ) {
-	webpackConfig.plugins.push( new PragmaCheckPlugin() );
+//	webpackConfig.plugins.push( new PragmaCheckPlugin() );
```

Comment:
Hmm, we can add analytics to the ignore array, but I think `state/themes/actions.js` should be ssr-ready from the start. Will take a look.